### PR TITLE
Add getting started guide for DA1469x

### PR DESCRIPTION
**Pull request by chrisc11**

### CODE REVIEW

File: docs/mcu/da1469x-sdk-guide.mdx [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+
+Memfault chunks can also easily be forwarded using
+[Web Bluetooth](https://web.dev/bluetooth/).
+
+To try it out, simply:
+
+- Open or install a recent version of the
+  [Google Chrome Browser](https://www.google.com/chrome/)
+- Make sure your BLE device is advertising
+- Navigate to the [Memfault MCS Web Bluetooth App](https://mflt.io/mcs), click
```
Comment (martijnthe) [inline docs/mcu/da1469x-sdk-guide.mdx:276]: Should we add this to the BLE "app" to the Chunks Debug page?

File: docs/mcu/da1469x-sdk-guide.mdx [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+Data can be extracted over bluetooth by implementing a MCS GATT Client in your
+gateway.
+
+Memfault chunks can also easily be forwarded using
+[Web Bluetooth](https://web.dev/bluetooth/).
+
+To try it out, simply:
+
+- Open or install a recent version of the
+  [Google Chrome Browser](https://www.google.com/chrome/)
```
Comment (noahp) [inline docs/mcu/da1469x-sdk-guide.mdx:274]: I found these instructions to be very helpful:
https://www.espruino.com/Quick+Start+BLE#with-web-bluetooth

Specifically, on linux I needed to enable this:
chrome://flags/#enable-experimental-web-platform-features

Comment (chrisc11) [reply docs/mcu/da1469x-sdk-guide.mdx:274]: ah, good to know thanks, it's not required on macOS for me, but looks like for some platforms support is still mixed https://www.chromestatus.com/feature/5264933985976320

**Comment by chrisc11** [comment]
> I only proof-read the text.
> 
> I've not tried the steps. I don't have a DA1469x board, but can at least try out the setup + compilation steps. Shall I do that?

proof read is great for now, thanks!

**Pull request merged**